tests/scripts/thread-cert/node.py

```diff
@@ -386,6 +387,73 @@ def get_border_routing_counters(self):
         logging.info(f'counters =  {counters} ')
         return counters
 
+    def _process_traffic_counters(self, counter):
+        return {
+            '4to6': {
+                'packets': counter[0],
+                'bytes': counter[1],
+            },
+            '6to4': {
+                'packets': counter[2],
+                'bytes': counter[3],
+            }
+        }
+
+    def _process_packet_counters(self, counter):
+        return {'4to6': {'packets': counter[0]}, '6to4': {'packets': counter[1]}}
+
+    def nat64_set_enabled(self, enable):
+        return self.call_dbus_method('io.openthread.BorderRouter', 'SetNat64Enabled', enable)
+
+    @property
+    def nat64_state(self):
+        state = self.get_dbus_property('Nat64State')
+        return {'PrefixManager': state[0], 'Translator': state[1]}
+
+    @property
```

Comment:
Just curious what's the advantage of making it a property?

Reply:
no special reasons

if there are any:
- these values are properties in dbus API
- these functions are just repacking the values from dbus